process/smartContract/builtInFunctions/esdtTransfer.go

```diff
@@ -187,6 +217,15 @@ func getESDTDataFromKey(
 	return esdtData, nil
 }
 
+func (e *esdtTransfer) setIsPayable(isPayableHandler process.IsPayableHandler) error {
```

Comment:
payableHandler process.PayableHandler is enough

Reply:
done